src/Microsoft.ML/Models/OneVersusAll.cs

```diff
@@ -13,10 +13,15 @@ namespace Microsoft.ML.Models
     public sealed partial class OneVersusAll
     {
         /// <summary>
-        /// Create OneVersusAll multiclass trainer.
+        /// One versus all learner (also in known as One vs the rest) is a multiclass learner with
+        /// strategy to fit one classifier per class. It picks each class and trains provided binary learner
+        /// against all the other classes. 
+        /// See <a href="https://en.wikipedia.org/wiki/Multiclass_classification"/> (One-vs.-rest) section
         /// </summary>
         /// <param name="trainer">Underlying binary trainer</param>
         /// <param name="useProbabilities">"Use probabilities (vs. raw outputs) to identify top-score category</param>
+        /// See <a href="https://github.com/dotnet/machinelearning/blob/master/test/Microsoft.ML.Tests/Scenarios/IrisPlantClassificationTests.cs"/>
+        /// for an example on how to train a OVA model.
```

Comment:
>a OVA [](start = 43, length = 5)

Hmmm. Is the expectation that they will say "oh-vee-ay" or "one versus all?" If the former case I'd prefer "an." I wonder what style guide we should adhere to for this sort of thing...